README.md

```diff
@@ -65,6 +65,29 @@ In the `superbol` field past the path to the `superbol` executable.
 
 You can check the documentation on using the extension on [this page](https://ocamlpro.github.io/superbol-vscode-platform/sphinx).
 
+### GNU/Emacs mode
```

Comment:
Just for the record : we should avoid adding this kind of technical documentation in the README.md file. Technical documentation should go to the Sphinx doc, while the README.md is just an entry point describing the project and its features, and linking to the Sphinx documentation for more technical details.

Reply:
Actually, as it is now, I find it quite helpful for the initial setup.

Reply:
Now, I think the fact that we can debate the place for the documentation means the GNU/Emacs mode itself does not fit in this particular repository very well. Wouldn't it be better to have a (however small) separate standalone `superbol-mode` repository with just the `README.md` (as with many other modes for Emacs) ?

Reply:
A minimal repo with the emacs folder and the README.md seems most useful.
The "common" parts of the LSP could be a git external set to a fixed revision of this repo that was tested against emacs.